rids/config.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Jigsaw Operations LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Configuration parsing and validataion.
+
+The config format may change, certainly its contents will.  This is where
+type-checking and consistency checks go.
+"""
+
+import json
+
+
+def parse_file(file_path):
```

Comment:
Not for this PR, but we should consider adding type definitions. Python without them is a nightmare.